tools/mpremote/mpremote/main.py

```diff
@@ -27,14 +28,14 @@
     do_connect,
     do_disconnect,
     do_edit,
+    do_eval,
+    do_exec,
     do_filesystem,
     do_mount,
-    do_umount,
-    do_exec,
-    do_eval,
-    do_run,
     do_resume,
+    do_run,
     do_soft_reset,
+    do_umount,
```

Comment:
please don't do any reformatting when it's not related to the commit (reformatting can be done separately, at least in a separate commit)

Reply:
IIRC the formatting was added by the recommended pre-commit hook, as apparently the original code was not up to spec.
makes sense to to seperate this out in a  commit, ( or I can remove this , but then the PR will fail again on CI)